.github/workflows/commentOnNativeFileChanged.yml

```diff
@@ -1,7 +1,7 @@
 name: Comment on native files changed
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
I don't love `pull_request_target`, it has special rules and we should double check them here: https://github.com/Expensify/App/blob/main/.github/workflows/README.md#security-rules-

Reply:
Also - I wonder if we can remove this workflow with this upcoming PR: https://github.com/Expensify/App/pull/55328

Reply:
Hmm that is good point, we might not need this so much with the workflow in the linked PR

Reply:
Yeah, it seems that this workflow won't have sense if the https://github.com/Expensify/App/pull/55328 is merged